src/firebase/userDb.js

```diff
@@ -0,0 +1,33 @@
+import { getAuth, signOut, signInWithPopup, GoogleAuthProvider, getAdditionalUserInfo } from 'firebase/auth';
+import { useAuthState } from 'react-firebase-hooks/auth';
+import { db } from './firebaseConfig';
+import { collection, addDoc, getDocs } from 'firebase/firestore';
+
+export function getAuthUser() {
+	const auth = getAuth();
+	const [user, loading] = useAuthState(auth);
+	return [user, loading];
+}
+
+// Todo
```

Comment:
i'm not a *huge* fan of merging code that isn't actually relevant to us into main, would rather it not be merged at all, but at least it has todo comments so i'm okay with it if you don't wanna cut it

Reply:
I have this file because I want it be a template for our code related to database. Are you saying it would be better to not have this file merged if it is incomplete?